src/scenes/PosterPage/PosterPage.tsx

```diff
@@ -44,7 +46,8 @@ export default function PosterPage() {
           <TitleM>2022 Community Poster</TitleM>
           <StyledParagraph>
             <BaseM>
-              Thank you for being a member of Gallery. Celebrate our new brand with us by signing
+              Thank you for being a member of Gallery. Celebrate our{' '}
```

Comment:
{' '} is a hack to keep a trailing space on the line so that we render a space before the link component